src/sentry/static/sentry/app/components/events/interfaces/stacktraceLink.tsx

```diff
@@ -194,17 +206,9 @@ class StacktraceLink extends AsyncComponent<Props, State> {
   }
 
   renderMatchNoUrl() {
-    const {config} = this.match;
-    const {organization} = this.props;
     const text = this.errorText;
-    const url = `/settings/${organization.slug}/integrations/${config?.provider.key}/${config?.integrationId}/?tab=codeMappings`;
     return (
-      <CodeMappingButtonContainer columnQuantity={2}>
-        {text}
-        <Button onClick={() => this.onReconfigureMapping()} to={url} size="xsmall">
```

Comment:
Also @NisanthanNanthakumar if we remove this we no longer have any idea of people tried to set up their configuration again, and if they were successful. I think in the case where we prompt them to auto set up their rule and it doesn't work for some reason, they now have no way to go to fix it, assuming that they are setting this up for the first time and don't know where the code mapping settings are